release-team/role-handbooks/communications/release-blog.md

```diff
@@ -48,6 +48,9 @@ This outline can be used as reference for writing up the release blog. The follo
     * Kubernetes blog
 
 ## Latest Release Blogs as Reference
```

Comment:
There is also v1.27 blog available: https://kubernetes.io/blog/2023/04/11/kubernetes-v1-27-release/